website/integrations/services/tandoor-recipes/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Tandoor Recipes
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Tandoor Recipes
+
+> Tandoor Recipes is an easy-to-use, powerful self-hosted food recipe manager.
+>
+> -- https://docs.tandoor.dev/
+
+Tandoor Recipes uses Django Allauth as the authentication engine. It supports OpenID Connect.
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `tandoor.company` is the FQDN of the Tandoor Recipes install.
+-   `authentik.company` is the FQDN of the authentik install.
+
+### Provider
+
+Create a Provider in authentik with the following parameters:
+
+-   Client type
+
+    Confidential
+
+
+-   Client ID
+
+    Take note of the Client ID.
+
+-   Client Secret
+
+    Take note of the Client Secret.
+
+-   Redirect URIs/Origins (RegEx)
+
+    https://tandoor.company/accounts/authentik/login/callback/
```

Comment:
This should be `https://tandoor.company/accounts/oidc/authentik/login/callback/` (note the added 'oidc').

Reply:
Were you able to fix the issue with linking existing local accounts to OIDC accounts?

Reply:
Yes. I configured it with Authentik as per the instructions in this PR (with the change as per my previous comment in the callback URL). I logged in as my local Tandoor user, went into user settings and there was a button to connect a social account. I did so, it went through Authentik's authorisation flow, and redirected me back to Tandoor, at which time it told me it was successfully linked. I've since logged out and back in without issue.

Reply:
figured out that I already got the 'oidc' part in my CallbackURL but it is still erroring out with Server Error 500 (ofc most recent version, both)

Reply:
I found the issue :/

I had to convert the PSQL encodging from SQL_ASCII to UTF8 on tandoor and after this my account was able to be connected with authentik

Reply:
Ok I think i figured it out. I had both configured - OIDC within the Django admin panel & as environment variables. I guess it was related to previous experiments. However it worked fine for the last 10 months. 
Now I removed the config from Django admin and exclusively configured docker env variables.
I detached the identities from the old OIDC provider and linked it again - now it is working again.